internal/ingress/controller/parser/ingressrules_test.go

```diff
@@ -0,0 +1,70 @@
+package parser
+
+import (
```

Comment:
The commit message of commit 18c51d34f28a1bec39a647af05877a6548bd1a78 has the following body:

```
This reverts commit b620751b8d49e854a7fb266b1f645053a1c095ea.
```

I _think_ this is incorrect.

Reply:
Fixed